### ISSUE TITLE
Removing Comments Section

### ISSUE THREAD

**Issue by kylebellamy** [jarednova/timber#638]
I inherited a site from another designer that did it in Timber. Since reading about it I intend to use it myself but I need some quick help as I don't have the time to find this info. But I am searching!

I need to remove the comments section on posts. Here is the code from the PHP file:

<?php
/**
 * The Template for displaying all single posts
 *
 * Methods for TimberHelper can be found in the /functions sub-directory
 *
 * @package  WordPress
 * @subpackage  Timber
 * @since    Timber 0.1
 */

$context = Timber::get_context();
$post = Timber::query_post();
$context['post'] = $post;
$context['wp_title'] .= ' - ' . $post->title();

$context['comment_form'] = TimberHelper::get_comment_form();

if (post_password_required($post->ID)){
	Timber::render('single-password.twig', $context);
} else {
	Timber::render(array('single-' . $post->ID . '.twig', 'single-' . $post->post_type . '.twig', 'single.twig'), $context);
}

Thanks for any help you can give.

**Comment by kylebellamy** [comment]
Ok so removing the comment_form line gets rid of the form but the title is still there in the page. Can't locate which php file assigns that title on the page. I looked in timber-posts but not sure if it's created there.

**Comment by gustavolanzas** [comment]
Kyle - if the comments only show up on the single post page, you'll just need to find the matching "single-post.twig" file. In there, look for where the title and comment form content is being rendered. Just comment out the parts you don't want.

**Comment by kylebellamy** [comment]
As it turns out, I just got access to ftp from the owner and located all the twig files. Looks a lot like Python Jira which I use everyday so once I determined that it was just a matter of locating what you pointed me to.

Now that I know that, I am definitely going to look into this template get system. Thanks for the push in the right direction.

Kyle Bellamy




On Mon, Aug 24, 2015 at 3:52 PM -0700, "Gustavo Lanzas" <notifications@github.com<mailto:notifications@github.com>> wrote:


Kyle - if the comments only show up on the single post page, you'll just need to find the matching "single-post.twig" file. In there, look for where the title and comment form content is being rendered. Just comment out the parts you don't want.

-
